mobilitydb/sql/general/CMakeLists.txt

```diff
@@ -22,10 +22,10 @@ SET(LOCAL_FILES
   032_temporal_boxops
   034_temporal_posops
   036_tnumber_distance
-  037_temporal_similarity
-  038_temporal_aggfuncs
-  040_temporal_waggfuncs
-  042_temporal_gist
+  038_temporal_similarity
+  040_temporal_aggfuncs
+  042_temporal_waggfuncs
+  044_temporal_gist
   044_temporal_spgist
```

Comment:
Is the identical numbering for temporal_gist and temporal_spgist on purpose or was that a mistake?
I believe it might be better to keep them distinct (ex. 043 and 044).